tests/test_geohash.py

```diff
@@ -0,0 +1,66 @@
+import pytest
+import numpy as np
+from numpy.testing import assert_array_equal
+import pandas as pd
+from pandas.testing import assert_index_equal
+from pygeohash import encode
+from dask_geopandas.geohash import _calculate_mid_points
+from dask_geopandas import from_geopandas
+import geopandas
+from shapely.geometry import Point, LineString, Polygon
+
+
+@pytest.fixture
+def geoseries_points():
+    p1 = Point(1, 2)
+    p2 = Point(2, 3)
+    p3 = Point(3, 4)
+    p4 = Point(4, 1)
+    return geopandas.GeoSeries([p1, p2, p3, p4])
+
+
+@pytest.fixture
+def geoseries_lines():
+    l1 = LineString([(0, 0), (0, 1), (1, 1)])
+    l2 = LineString([(0, 0), (1, 0), (1, 1), (0, 1)])
+    return geopandas.GeoSeries([l1, l2] * 2)
+
+
+@pytest.fixture()
+def geoseries_polygons():
+    t1 = Polygon([(0, 3.5), (7, 2.4), (1, 0.1)])
+    t2 = Polygon([(0, 0), (1, 1), (0, 1)])
+    sq1 = Polygon([(0, 0), (1, 0), (1, 1), (0, 1)])
+    sq2 = Polygon([(0, 0), (1, 0), (1, 2), (0, 2)])
+    return geopandas.GeoSeries([t1, t2, sq1, sq2])
+
+
+def geohash_dask(geoseries):
```

Comment:
This doesn't really work as a test since it uses the same code.

Reply:
yes, I need to update this. I was planning on using pygeohash as the reference for the numpy version but I think the numpy version will become another one of the follow-ups.

Reply:
since I have now committed the numpy version, this now works ok.